features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/BroadcastEventProcessor.java

```diff
@@ -45,6 +46,9 @@
  */
 final class BroadcastEventProcessor implements EventListener {
     private static final Logger LOG = LoggerFactory.getLogger(BroadcastEventProcessor.class);
+    
+    @Autowired
+    private SyslogdIPMgrDaoImpl syslogdIPMgrDaoImpl;
```

Comment:
Please change this to the interface (SyslogdIPMgr) since we want to be able to inject any class that implements the interface.